packages/twenty-front/src/modules/object-record/record-table/record-table-cell/components/RecordTableCellContainer.tsx

```diff
@@ -142,8 +150,16 @@ export const RecordTableCellContainer = ({
             [styles.cellBaseContainerSoftFocus]: hasSoftFocus,
           })}
         >
-          {isInEditMode && (
-            <RecordTableCellEditMode>{editModeContent}</RecordTableCellEditMode>
+          {isRecordTableInitialLoading ? (
+            <RecordTableCellContainerSkeletonLoader
+              skeletonWidth={skeletonWidth}
+            />
+          ) : (
+            isInEditMode && (
+              <RecordTableCellEditMode>
+                {editModeContent}
+              </RecordTableCellEditMode>
+            )
```

Comment:
Hey @lucasbordeau 
Here we need to render the individual loader for each cell during initial table loading, so we can't just move this loader to a new component (RecordTableLoading). However, this only shows during the initial load so shouldn't cause a performance issue.

What do you think?!

Reply:
Turns out this has cause a performance hit of about 0.01s, huh! 😅

Reply:
Turns out this has caused a performance hit of about 0.01s

Reply:
working on moving the loading logic out of the `RecordTableCell`